contracts/cw3-fixed-multisig/src/state.rs

```diff
@@ -318,6 +404,95 @@ mod test {
         assert!(check_is_passed(quorum, passing, 16, false));
     }
 
+    #[test]
+    fn proposal_rejected_quorum() {
+        let quorum = Threshold::ThresholdQuorum {
+            threshold: Decimal::percent(50),
```

Comment:
Let's use threshold = Decimal::percent(60) so we can test the real case. Then fix the logic to match